jekyll/_config.yml

```diff
@@ -155,6 +155,12 @@ defaults:
     values:
       suggested: false
 
+  # Disable sitemap by default (only want in prod)
+  - scope:
+      path: ""
+    values:
+      sitemap: false
+
```

Comment:
Just curious, why do we want to disable sitemaps here only to enable it in `_config_production.yml` for prod?

Reply:
Generating the sitemap in local takes between 12 and 15 seconds for **each** rebuild. We only need the sitemap when we are building prod so this is why I disabled it